kernel/src/main/scala/cats/kernel/CommutativeGroup.scala

```diff
@@ -0,0 +1,17 @@
+package cats.kernel
+
+import scala.{ specialized => sp }
+
+/**
+ * An abelian group is a group whose operation is commutative.
```

Comment:
Minor: we might want something like "A `CommutativeGroup` (AKA abelian group) is a group whose `combine` operation is commutative".